frontend/src/pages/main/home/client/clientHome/ClientHome.tsx

```diff
@@ -2,37 +2,23 @@ import React from 'react';
 import {Box} from "@mui/material";
 import {Titles} from "../../../../../common/constansts/titles";
 import Balance from "../balance/Balance";
-import ScanPayEntry from "../scanPayEntry/ScanPayEntry";
-import TopUpEntry from "../topUpEntry/TopUpEntry";
-import TransactionsEntry from "../transactionsEntry/TransactionsEntry";
 import Title from "../../../../../components/title/Title";
-import {useNavigate} from "react-router-dom";
 import {Paths} from "../../../../../common/constansts/paths";
 import "./styles.css";
+import HomeEntry from "../homeEntry/HomeEntry";
+import {Content} from "../../../../../common/constansts/content";
+import {Icons} from "../../../../../common/constansts/icons";
 
 const ClientHome: React.FC = () => {
-    const navigate = useNavigate();
-
-    const handlePaymentClick = () => {
-        navigate(Paths.PAYMENT);
-    }
-
-    const handleTopUpClick = () => {
-        navigate(Paths.TOP_UP);
-    }
-
-    const handleHistoryClick = () => {
-        navigate(Paths.TRANSACTIONS);
-    }
 
     return (
         <Box className="client-home">
             <Title title={Titles.HOME}/>
             <Balance/>
             <Box className="components-container">
-                <ScanPayEntry handlePaymentClick={handlePaymentClick}/>
-                <TopUpEntry handleTopUpClick={handleTopUpClick}/>
-                <TransactionsEntry handleTransactionsClick={handleHistoryClick}/>
+                <HomeEntry content={Content.SCAN_PAY} path={Paths.PAYMENT} iconName={Icons.CENTER_FOCUS_WEAK}/>
```

Comment:
Alternative solution.

Change props to accept react component
```typescript
export interface HomeEntryProps {
    content: string,
    path: string,
    icon: ReactNode,
}
```

then pass icon component
```typescript
<HomeEntry content={Titles.TRANSACTIONS} path={Paths.TRANSACTIONS} icon={<AddCardIcon className="icon"/> color="primary"}/>`
``` 
in home entry use it like this
 ```typescript
<Box className="home-components-info">
                        {icon}
                        <Typography variant="h6">
                            {content}
                        </Typography>
                    </Box>
``` 

I don't like it because we need to repeat props on each icon: className, color and so on